tests/pages/__snapshots__/index.test.tsx.snap

```diff
@@ -324,26 +324,7 @@ exports[`Exemplar react-test-renderer Snapshot test renders homepage unchanged 1
                   <div
                     className="info-entry"
                   >
-                    4/29/2021
-                  </div>
-                </div>
-              </div>
-              <div
-                className="col-6"
-              >
-                <div />
-                <div
-                  className="info"
-                >
-                  <div
-                    className="info-label"
-                  >
-                    Extension Type
-                  </div>
-                  <div
-                    className="info-entry"
-                  >
-                    Pandemic Emergency Unemployment Compensation (PEUC) - Tier 2 Extension
```

Comment:
Did we expect this to go away? 👀

Reply:
Yes. I switched the default `programType` to one that doesn't have an extension type.